src/test/ui/error-codes/E0603.stderr

```diff
@@ -1,8 +1,11 @@
 error[E0603]: constant `PRIVATE` is private
   --> $DIR/E0603.rs:6:17
    |
+LL |     const PRIVATE: u32 = 0x_a_bad_1dea_u32;
```

Comment:
I think the order of these two parts (`this constant is private` and `constant PRIVATE declared here`) should potentially be reversed, as the latter is the actual issue.